samples/cache4k-kmm/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Sat Jan 09 13:19:55 CET 2021
+distributionBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
```

Comment:
I assume 6.8 doesn't work?

Reply:
It was failing yesterday, I tried today and it is working. Maybe some corrupted cache file